go/client/cmd_simplefs_copy.go

```diff
@@ -57,6 +57,10 @@ func NewCmdSimpleFSCopy(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 				Name:  "rev",
 				Usage: "specify a revision number for the KBFS folder of the source paths",
 			},
+			cli.StringFlag{
+				Name:  "time",
+				Usage: "specify a time for the KBFS folder of the source paths (eg \"7/27/18 22:05\")",
```

Comment:
If you decide to implement my feedback [here](https://github.com/keybase/kbfs/pull/1706#discussion_r205625168), the example dates in this file should be a non-ambiguous one too.